src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodsServiceSpec.scala

```diff
@@ -52,7 +50,7 @@ class MethodsServiceSpec extends FreeSpec with ScalaFutures with ScalatestRouteT
       "MethodNotAllowed error is returned" in {
         Put("/methods") ~> sealRoute(routes) ~> check {
           status should equal(MethodNotAllowed)
-          responseAs[String] === "HTTP method not allowed, supported methods: GET"
+          errorReportCheck("Agora", MethodNotAllowed)
```

Comment:
My thought was to write these as unit/integration agnostic, so they could be migrated to integration tests as appropriate.